rust/xet_error/_git/logs/HEAD

```diff
@@ -1 +1 @@
-0000000000000000000000000000000000000000 e9ea67c7e251764c3c2d839b6c06d9f35b154647 Hoyt Koepke <hoytak@xethub.com> 1701979145 -0800	clone: from github.com:dtolnay/thiserror.git
+0000000000000000000000000000000000000000 e9ea67c7e251764c3c2d839b6c06d9f35b154647 Hoyt Koepke <hoytak@xethub.com> 1701979145 -0800	clone: from github.com:dtolnay/xet_error.git
```

Comment:
Not sure this rename makes sense?

Reply:
The issue here is that I took a source drop of the thiserror package as I needed to modify its internals to create the callback.  To effectively allow us to update thiserror in the future, I committed the .git/ folder as _git.  If we need to update the code in the future, we can rename that directory to do this effectively.  

However, when we specified thiserror locally by that name, cargo chose this one over the external version, including for all the dependent packages like opentelemetry.  This is where the error comes from, hence the rename to xet_error

Reply:
I see, I think we might specifically want to preserve `clone: from github.com:dtolnay/thiserror.git` in this file and not replace it with xet_error?

Reply:
True.  Good call.